src/Instrumentation/PDO/src/PDOInstrumentation.php

```diff
@@ -329,4 +377,29 @@ private static function isDistributeStatementToLinkedSpansEnabled(): bool
 
         return filter_var(get_cfg_var('otel.instrumentation.pdo.distribute_statement_to_linked_spans'), FILTER_VALIDATE_BOOLEAN, FILTER_NULL_ON_FAILURE) ?? false;
     }
+
+    private static function isSqlCommenterEnabled(): bool
+    {
+        if (class_exists('OpenTelemetry\SDK\Common\Configuration\Configuration')) {
+            return Configuration::getBoolean('OTEL_PHP_INSTRUMENTATION_PDO_SQL_COMMENTER', false);
+        }
+
+        return filter_var(get_cfg_var('otel.instrumentation.pdo.sql_commenter'), FILTER_VALIDATE_BOOLEAN, FILTER_NULL_ON_FAILURE) ?? false;
+    }
+
+    private static function appendSqlComments(string $query, bool $withTraceContext): string
```

Comment:
Isn't there also a requirement to prepend for some DBMS?

Reply:
Sure. Done!